saleor/graphql/account/mutations/authentication/request_password_reset.py

```diff
@@ -79,12 +83,12 @@ def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
         email = data["email"]
         redirect_url = data["redirect_url"]
         user = cls.clean_user(email, redirect_url)
-        channel_slug = data.get("channel")
+        channel = data.get("channel")
 
-        channel_slug = clean_channel(
-            channel_slug, error_class=AccountErrorCode, allow_replica=False
-        ).slug
-        channel_slug = validate_channel(channel_slug, error_class=AccountErrorCode).slug
+        if channel:
+            channel_slug = validate_channel(channel, error_class=AccountErrorCode).slug
+        else:
+            channel_slug = ""
```

Comment:
Suggestion (non-blocking):
```suggestion
        channel_slug = validate_channel(channel, error_class=AccountErrorCode).slug if channel else ""
```